javascript/index.js

```diff
@@ -35,7 +35,7 @@ const portfolioCards = [
     description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
```

Comment:
- [ ] Please the headline section doesn't match the design the background and the text are not aligned properly.
![image](https://user-images.githubusercontent.com/74744787/219766421-4142b23d-2472-4253-9cde-32c03b264a1b.png)

suggestions: 
1- remove this absolute positioning, I think HTML flow is enough to structure your HTML elements 
![image](https://user-images.githubusercontent.com/74744787/219766517-82a3f0a5-39d0-413a-98dd-c70672488640.png)
2- add some margin to the headline section so you have the right space between the toolbar and the headline look how it looks now 
![image](https://user-images.githubusercontent.com/74744787/219766970-59548829-2c7c-4986-a9b8-ba6af31635d3.png)